### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by kinglau66** [microsoft/cascadia-code#676]
# Description of the new feature/enhancement (with images if possible)

Inifinite arrow is nice but i find it distracting for my everyday work.
possible for a stylistic set to disable or config the length of infinite arrow? 
eg 'infi3' only allow infinite up to 3 character ligature
![image](https://user-images.githubusercontent.com/15945904/217983668-ff4e8faf-bdee-4ee3-a7f4-ed8a1b28af84.png)

**Comment by aaronbell** [comment]
Yes, I am thinking of rejigging how the coding ligatures are implemented and this would likely be implemented as well. Thanks
